### DISCUSSION TITLE
new setup not ok

### DISCUSSION THREAD

**Discussion by alero63** [jomjol/AI-on-the-edge-device#2685] [General]
HI,

i have reconfigure of my powermeter. I create a new reference image create new marker and i reset all digs and create new ones. As the pic shows everything looks good.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/02ba160b-6d77-455e-b7aa-22e47a0435b7)
But if i look at recognition i see that.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/f8bc3132-45ee-49d5-ae87-0feab478b82a)
what did i do wrong?

**Comment by friedpa** [comment]
The Alignment Marks are not working correctly. Please show your Reference Image with the Marks. The recommendation is to spreadthem out as far as possible (just use a black sticker on the upper left and the lower right corner, as better the contrast, as better the result.

**Comment by alero63** [comment]
hi Pual,
thanks for your fast response.
here is my Refeerrence Picture.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/8422687b-f2e5-444e-bf0d-be6fb751ebc3)
What do you mean with black Sticker in the ...?
Just put a black Sticker to the front (Glas)  of the powermeter?

**Comment by alero63** [comment]
Hi, i find a way. i put some black stripes on top and button of the powermeter and use a tape which cover the ligth to 50%.
Now it works.

Thanks for your Help

**Comment by alero63** [comment]
thanks again
